guides/source/layouts_and_rendering.md

```diff
@@ -151,7 +151,7 @@ render template: "products/show"
 
 #### Wrapping it up
 
-The above three ways of rendering (rendering another template within the controller, rendering a template within another controller, and rendering an arbitrary file on the file system) are actually variants of the same action.
+The above two ways of rendering (rendering another template within the controller and rendering a template within another controller) are actually variants of the same action.
```

Comment:
Since we're editing this line anyway, what do you think about clarifying the language a bit?

```suggestion
The above two ways of rendering (rendering the template of another action in the same controller, and rendering the template of another action in a different controller) are actually variants of the same operation.
```